web3/queries/votes/getAssertionClaim.ts

```diff
@@ -7,7 +7,7 @@ export async function getAssertionMadeEvents(
 ) {
   const currentBlock = await instance.provider.getBlockNumber();
   const filter = instance.filters.AssertionMade(assertionId);
-  return instance.queryFilter(filter, currentBlock - 100000);
+  return instance.queryFilter(filter, currentBlock - 1000000);
```

Comment:
Will this not exceed block limits on some chains?

Reply:
most likely yes, ill need to make a note to fix this later.  unfortunately on story with the current range, we cant find the claim, so had to bump it up. fortunately it still works

Reply:
made an issue for it: https://linear.app/uma/issue/UMA-2867/fix-assertion-claim-lookback-to-page-in-shorter-ranges